.github/workflows/test.yml

```
@@ -11,12 +11,32 @@ on:
       - main
 
 jobs:
+  define-sessions:
+    runs-on: ubuntu-latest
+    outputs:
+      sessions: ${{ steps.set-sessions.outputs.sessions }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'      
+      - run: pip install nox
+      - name: List and set Nox sessions matrix
+        id: set-sessions
+        run: |
+          sessions=$(nox --list --tag "test" --json | jq -r '. | map("\"" + .session + "\"") | join(",")')
+          echo "sessions=[$sessions]" >> $GITHUB_OUTPUT
```

Can we simplify this to the following? Grabbing the `session` field of each object and returning valid JSON directly?

```suggestion
          sessions=$(nox --list --tag "test" --json | jq '[ .[] | .session ]')
          echo "sessions=$sessions" >> $GITHUB_OUTPUT
```

---

Oh good call. I found I had to add the `--compact-output` flag as well, or else I got [an error](https://github.com/grahamalama/fourkeys/actions/runs/11114716192/job/30881735319)

---

👍🏻